### ISSUE TITLE
[4.8 beta] Settings guide blanked out words

### ISSUE THREAD

**Issue by printingotb** [Ultimaker/Cura#8675]
**Application version**
4.8 beta

**Platform**
win7

**Printer**
any

**Reproduction steps**
1. Enable "show articles in settings tooltips"
2. Open Settings visibility in preferences, hover the cursor over settings.

**Screenshot(s)**
![Settign descriptions](https://user-images.githubusercontent.com/32172100/97855850-ac6b1500-1cfb-11eb-9ac2-1c5c61e90b10.png)

**Actual results**
words blanked out

**Expected results**
not blanked out

**Additional information**
Side-note: the text under the left picture of "Connect Top/bottom Polygons says "Naive concentric pattern", should it be "native"?

And by the way, is reporting these kind of bugs overkill?

**Comment by Asterchades** [comment]
> And by the way, is reporting these kind of bugs overkill?

No, but reporting them here isn't appropriate. The Settings Guide isn't part of Cura - it's an extension.

You'll want to make a report on the repo responsible for the extension: https://github.com/Ghostkeeper/SettingsGuide

**Comment by printingotb** [comment]
I see, how do I delete this post?

**Comment by Asterchades** [comment]
Honestly? I'm not entirely sure that you can. But you can close the issue, which is mostly the same thing in the grand scheme of things. There should be a button down with the one you use to post a comment allowing you to do exactly that (either by itself or with a response).

**Comment by printingotb** [comment]
Found it :sweat_smile:
